examples/with-tailwindcss/postcss.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   plugins: [
     'tailwindcss',
-    process.env.NODE_ENV === 'production'
+      ...(process.env.NODE_ENV === 'production'
```

Comment:
Seems that this will spread the options object of purgecss into the main array 🤔

Reply:
Oh yea missed that, thanks 🤦‍♂